### ISSUE TITLE
Separate tests from start-nightfall script

### ISSUE THREAD

**Issue by signorecello** [EYBlockchain/nightfall_3#403]
As of now, `start-nightfall` is a bit of everything. We use it to deploy with a built-in ganache, we use it to deploy to localhost node, we use it to deploy a wallet, we use it to test stuff... I believe it will get very confusing very fast if we don't separate stuff.

Some suggestions on how to accomplish this:

- Remove the `-wt` flag
- Rename the `wallet-test` docker service to just `wallet`
- Make the gh actions wait for a 200 status code instead of a fixed 1500s sleep because it may not be enough for the containers to start (I suggest using [something like this](https://github.com/marketplace/actions/wait-for-http-responses))
- Change gh actions to run the tests on `wallet` instead of `wallet-test` using something like `docker exec`

So basically we only use `start-nightfall` to _start_ stuff, and the gh actions will use `docker exec` or similar stuff to run the tests inside the containers.

**Comment by ChaitanyaKonda** [comment]
#490 solves this
